apps/native-component-list/src/components/FunctionDemo/index.types.ts

```diff
@@ -30,7 +30,7 @@ export type EnumParameter = Parameter & {
   type: 'enum';
   values: {
     name: string;
-    value: undefined | string | number | Tuple;
+    value: undefined | string | number | Tuple | any;
```

Comment:
```suggestion
    value: any;
```

We can just use `any` here, if we need to add it anyway.